README.md

```diff
@@ -14,33 +14,22 @@
 
 ## <a name="overview"></a>Overview
 
-### Prime Directive
+The Nerdery’s perspective is that code exists to solve a problem. We favor solutions which are pragmatic and innovative over adherence to any one approach, abstraction, or toolset. Because the front-end engineering ecosystem is constantly evolving, a set of guidelines for working within it must continue to be relevant even as specific implementations change. We prioritize interoperability when choosing solutions, and use our guiding principles to write code that is resilient and easy to maintain. We also look to the practices of the front-end community as a whole to create a system of universal consistency.
 
-The standards herein are a guide to keep developers at the Nerdery interoperable amongst each other and able to produce the highest quality code given the timeline and budget constraints of a project.
+This is a living document and is reflective of the opinions and expertise of The Nerdery’s front-end engineering group. However, the discussion of, contribution to, and evolution of this document is reliant on the front-end community both inside and outside of The Nerdery.
 
-For a successful project, always favor solutions that benefit the project and end users over producing "perfect code".
+### Guiding Principles
```

Comment:
The guiding principles have been updated to share a voice and be entirely consumer centric (not a guidepost for the updates being made themselves). this is reflected in 59c6f1b.